src/sass/components/_header.scss

```diff
@@ -116,7 +116,7 @@ header {
     background-color: #ffffff;
 
     .container > .row {
-      height: 84px;
+      /*height: 84px;*/
```

Comment:
jeśli jest to zakomentowane, to czy w takim razie potrzebujemy w ogóle tą regułę?